### PR TITLE
fix: run dependencies in sequenced order

### PR DESCRIPTION
**Pull request by ryanio**
#### What does it do?
Fixes `clef` app dependencies and ensures they run in sequenced order so clef can start first then geth (with signer clef option)

---

Closes https://github.com/ethereum/grid/issues/462

### CODE REVIEW

**Comment by marcgarreau** [comment]
Not sure if appropriate scope for this PR, but starting the Clef app successfully starts Clef, then Geth, but then no app until another click: https://imgur.com/PsUcsry.

File: grid_apps/AppManager.js [hunk 1]
```diff
@@ -162,6 +162,7 @@ class AppManager extends EventEmitter {
 
     const { dependencies } = app
     if (dependencies) {
+      var sequence = Promise.resolve()
```
Comment (marcgarreau) [inline grid_apps/AppManager.js]: this is an unusual pattern, so may be worth adding the stackoverflow link as a comment above this for context

Comment (ryanio) [reply grid_apps/AppManager.js]: added, thanks

File: grid_apps/AppManager.js [hunk 2]
```diff
@@ -162,49 +162,55 @@ class AppManager extends EventEmitter {
 
     const { dependencies } = app
     if (dependencies) {
+      // Execute dependneices one at a time with a sequence
+      // as inspired by https://stackoverflow.com/a/36672042
```
Comment (PhilippLgh) [inline grid_apps/AppManager.js]: https://stackoverflow.com/a/53050379/5675137

Comment (ryanio) [reply grid_apps/AppManager.js]: thanks, will swap.

**Comment by ryanio** [comment]
just tried this on master and it worked 👍

**Pull request closed without merging**